etc/config/racket.amazon.properties

```diff
@@ -0,0 +1,9 @@
+compilers=&racket
+defaultCompiler=racket86
+
+group.racket.compilers=racket86
+group.racket.isSemVer=true
+group.racket.baseName=Racket
```

Comment:
A `.groupName` key would also be needed.
Also, we're as of a few days ago, we're asking compiler groups to provide license info
group.racket.licensePreamble for custom text to be shown,
group.racket.licenseName for the license name,
group.racket.licenseLink for either a link to the license or other relevant URL.

All three are optional if you don't have anything for the preamble, for example

Reply:
Ah thanks, I believe I've added these extra bits correctly. I left out the preamble, as it doesn't seem needed in this case.